tests/test_property.py

```diff
@@ -89,6 +174,7 @@ def test_to_graph_should_return_blank_node() -> None:
         @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
 
         [] a modelldcatno:Property .
```

Comment:
```
[ a    modelldcatno:Property; 
  modelldcatno:hasType    [ a modelldcatno:ModelElement ]
] .
```

Reply:
Takk for tilbakemeldinger @NilsOveTen , da skal jeg se på disse! :-)